src/utils/time.ts

```diff
@@ -29,18 +21,16 @@ export const toMoment = (value?: moment.MomentInput): moment.Moment => {
  * @param  {String} date   the date to calculate number of days since
  * @return {Number} an integer of the number of days
  */
-export const daysSince = (date: string): number => {
-    const diff = toMoment().startOf('day').diff(toMoment(date).startOf('day'), 'days');
-    return !date ? 0 : diff;
-};
+export const daysSince = (date: string): number =>
+    toMoment().startOf('day').diff(toMoment(date).startOf('day'), 'days');
 
 /**
  * The below function is used to format the display the time given in minutes to hours and minutes
  * e.g. 90 minutes will be displayed as 1 hour 30 minutes
- * @param {number} minutes
+ * @param {number} minutes - The time in minutes (convert to epoch time before passing)
  * @returns {string} formatted time string e.g. 1 hour 30 minutes
  */
-export const formatTime = (minutes: number) => {
+export const formatTime = (minutes: number): string => {
```

Comment:
```suggestion
export const formatTime = (minutes: number => {
```
typescript will infer it

Reply:
ah yeah i know, it had just given me a small note so i just added it since we do it to our other functions 👀 

![Screenshot 2024-06-21 at 11 18 34 AM](https://github.com/deriv-com/p2p/assets/103412909/3c484c69-0dd2-49bc-8165-c147cf616b28)

Reply:
ahh ok then

Reply:
oh you can ignore this JSDOCs note 😄

Reply:
😆 alright, but i think its just fine for now, since we did it for all of the other functions 😛

Reply:
Ok then 😄